ar_exercise/exercise.rb

```diff
@@ -34,9 +34,17 @@ class Apartment < ActiveRecord::Base
 all_tenants = Tenant.all
 
```

Comment:
All the methods below are from ActiveRecord `.find` `.where` `.find_by`; these will not work since you've commented outlines 8-19. What've you written is correct, though. You would just have no way of knowing or testing if you comment out the line where the models inherit form ActiveRecord::Base